spec/duration.html

```diff
@@ -1434,8 +1426,10 @@ <h1>
         <dd>It adds the components of a second duration represented by _y2_ through _ns2_ to those of a first duration represented by _y1_ through _ns1_, and balances the duration relative to the given date _relativeTo_, to ensure that no mixed signs remain in the result. The result is returned as a Duration Record.</dd>
       </dl>
       <emu-alg>
-        1. Let _largestUnit1_ be ! DefaultTemporalLargestUnit(_y1_, _mon1_, _w1_, _d1_, _h1_, _min1_, _s1_, _ms1_, _mus1_).
-        1. Let _largestUnit2_ be ! DefaultTemporalLargestUnit(_y2_, _mon2_, _w2_, _d2_, _h2_, _min2_, _s2_, _ms2_, _mus2_).
+        1. Let _duration1_ be ! CreateDurationRecord(_y1_, _mon1_, _w1_, _d1_, _h1_, _min1_, _s1_, _ms1_, _mus1_).
+        1. Let _duration2_ be ! CreateDurationRecord(_y2_, _mon2_, _w2_, _d2_, _h2_, _min2_, _s2_, _ms2_, _mus2_).
+        1. Let _largestUnit1_ be ! DefaultTemporalLargestUnit(_duration1_).
+        1. Let _largestUnit2_ be ! DefaultTemporalLargestUnit(_duration2_).
```

Comment:
Hmm, this is not as obvious as it seems. CreateDurationRecord does a MV → float → MV dance. Are you sure none of these changes are affected?

Reply:
I corrected this to remain in MV domain with a later commit.